packages/client/hmi-client/src/services/dataset.ts

```diff
@@ -21,7 +21,7 @@ async function getAll(): Promise<Dataset[] | null> {
 	return response?.data ?? null;
 }
 
-async function getAllClimate(query: string): Promise<Dataset[] | null> {
+async function searchClimateDatasets(query: string): Promise<Dataset[] | null> {
```

Comment:
I prefer using the word `question` instead of `query`, it clarify that this is an English string written by a human instead of a software created query. If that makes sense?
```suggestion
async function searchClimateDatasets(question: string): Promise<Dataset[] | null> {
```

Reply:
A human searching something isn't always going to be phrased as a question though

Reply:
Not a question in the english grammar sense, but a query in english